site/gulpfile.js

```diff
@@ -6,10 +6,10 @@ elixir.config.assetsPath = 'source/_assets';
 elixir.config.publicPath = 'source';
 
 elixir(function(mix) {
-    var env = argv.e || argv.env || 'local';
+    var env = argv.e || 'local';
```

Comment:
I think we should keep the `argv.env` as that allows `--env=...` and just `-e=` for gulp.